host/xtest/xtest_test.h

```diff
@@ -98,6 +98,9 @@ extern const unsigned int gp_tta_tcf_singleinstance_ta_size;
 extern const char gp_tta_tcf_multipleinstance_ta[];
 extern const unsigned int gp_tta_tcf_multipleinstance_ta_size;
 
+extern const char tpm_log_test_ta[];
+extern const unsigned int tpm_log_test_ta_size;
```

Comment:
I can't find why these are needed?

Reply:
Hmmm... Forgot to remove them whilst refactoring